website/docs/Basic_operations/create.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 2
+---
+
+# Insert operation
+
+The insert operation adds a new document to a collection.
+
+## Insert a single document
+
+The `insertOne()` command is used to add a single document into a collection, using this syntax format:
+
+```sh
+db.collection.insertOne({field1: value1, field2: value2,.... fieldN: valueN})
+```
+
+The example below depicts how a single document can be added to a collection.
+
+```sh
+db.scientists.insertOne({name:{firstname: "Thomas", lastname: "Edison"}, born: 1847, invention: "lightbulb"})
+```
+
+If the operation is successful, you will get a response with acknowledged set to true, and the autogenerated ObjectId of the document.
+
+:::note
+If a collection does not exist, the insert command automatically creates one.
+:::
+
+## Insert multiple documents at once
+
+A collection can contain multiple documents.
+Using the `insertMany()` command, you can add multiple documents to a collection at once.
+
+```sh
+db.collection_name.insertMany([{document1}, {document2},... {documentN}])
+```
+
+The following example shows how to insert multiple documents into a collection:
+
+```sh
+db.scientists.insertMany([{name:{firstname: "Alan", lastname: "Turing"}, born: 1912, invention: "Turing Machine"},{name:{firstname: "Graham", lastname: "Bell"}, born: 1847, invention: "telephone"},{name:{firstname: "Ada", lastname: "Lovelace"}, born: 1815, invention: "computer programming"}])
```

Comment:
I wonder if we should format this code (e.g. make it a multi-line json with indents), so it's more readable for the readers of our article... Also, the way how whitespace is used (e.g. between `,` and `{`) is a bit inconsistent. I'd say we should do some automatic reformat of such code examples with an editor reformatting tool or `jq` or something like that.